go.sum

```diff
@@ -122,10 +122,12 @@ github.com/kr/pretty v0.1.0/go.mod h1:dAy3ld7l9f0ibDNOQOHHMYYIIbhfbHSm3C4ZsoJORN
 github.com/kr/pty v1.1.1/go.mod h1:pFQYn66WHrOpPYNljwOMqo10TkYh1fy3cYio2l3bCsQ=
 github.com/kr/text v0.1.0/go.mod h1:4Jbv+DJW3UT/LiOwJeYQe1efqtUx/iVham/4vfdArNI=
 github.com/lacework/conntrack v0.0.0-20200724052553-e16a10e68325/go.mod h1:yV2Weh5TLUf5Hhttml+lyLlwb9zIWUv3/ps0KnR8Utg=
+github.com/lacework/conntrack v0.0.0-20210812163929-c9e4ef49253e/go.mod h1:yV2Weh5TLUf5Hhttml+lyLlwb9zIWUv3/ps0KnR8Utg=
```

Comment:
two entries for the same conntrack?

Reply:
It seems to be there for other packages too.